Explorer/Assets/DCL/LOD/Components/SceneLODInfo.cs

```diff
@@ -108,7 +137,11 @@ public bool HasLOD(byte lodForAcquisition)
                    CurrentLODLevelPromise == lodForAcquisition;
         }
 
-
+        public void SetCurrentLODPromise(AssetPromise<AssetBundleData, GetAssetBundleIntention> promise, byte lodLevel)
```

Comment:
This function is redundant because as we spoke we can't convert AssetPromise to the property